src/RealtimeEffectPanel.cpp

```diff
@@ -412,6 +438,8 @@ namespace
          }
 
          auto& effectStateUI = RealtimeEffectStateUI::Get(*mEffectState);
+
+         effectStateUI.UpdateTrackData(*mTrack);
```

Comment:
Does this line properly belong to the earlier commit "Adds a class to manage stacked effects settings UI" ?

Reply:
I would say no

Reply:
Not sure I understood why.